### DISCUSSION TITLE
corpora.train -> path   value is not a valid path

### DISCUSSION THREAD

**Discussion by myeghaneh** [explosion/spaCy#6687] [Help: Coding & Implementations]
I have used this comment 

```
python -m spacy convert  .\data\train.json .\data\

to convert my json data (train-test-dev)

```
[["text bla bla bla .",
  {'entities': [[731, 737, 'ARC'],
    [757, 763, 'ARC'],
    [836, 839, 'ARC'],
    [1144, 1147, 'ARC'],
    [1381, 1384, 'ARC'],
    [1389, 1392, 'ARC'],
    [1485, 1488, 'ARC'],
    [394, 403, 'ZOD'],
    [925, 933, 'MODELL'],
    [911, 919, 'MODELL'],
    [872, 877, 'MODELL'],
    [307, 317, 'CELOB'],
    [294, 298, 'CELOB'],
    [448, 452, 'CELOB'],
    [685, 689, 'CELOB'],
    [282, 290, 'AST'],
    [355, 363, 'AST'],
    [463, 471, 'AST'],
    [537, 545, 'AST'],
    [718, 726, 'AST'],
    [819, 827, 'AST'],
    [994, 1002, 'AST'],
    [1164, 1172, 'AST'],
    [234, 245, 'METH'],
    [1109, 1120, 'METH'],
    [676, 679, 'METH']]}] ,...
]
```
to binary, and now I have train,dev,test.json files which is good 
then i used  this command 

```
python -m spacy train filled_config.cfg --paths.train  .\data\spacy.train
```
to train but it said that

```

=========================== Initializing pipeline ===========================
✘ Config validation error
corpora.train -> **path   value is not a valid path**

{'@readers': 'spacy.Corpus.v1', 'path': True, 'max_length': 2000, 'gold_preproc': False, 'limit': 0, 'augmenter': None}
```

here is my config file 


```

[paths]
train = null
dev = null
vectors = null
init_tok2vec = null

[system]
gpu_allocator = null
seed = 0

[nlp]
lang = "en"
pipeline = ["tok2vec","parser","ner","tagger"]
tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
disabled = []
before_creation = null
after_creation = null
after_pipeline_creation = null

[components]

[components.ner]
factory = "ner"
moves = null
update_with_oracle_cut_size = 100

[components.ner.model]
@architectures = "spacy.TransitionBasedParser.v1"
state_type = "ner"
extra_state_tokens = false
hidden_width = 64
maxout_pieces = 2
use_upper = true
nO = null

[components.ner.model.tok2vec]
@architectures = "spacy.Tok2VecListener.v1"
width = ${components.tok2vec.model.encode.width}
upstream = "*"

[components.parser]
factory = "parser"
learn_tokens = false
min_action_freq = 30
moves = null
update_with_oracle_cut_size = 100

[components.parser.model]
@architectures = "spacy.TransitionBasedParser.v1"
state_type = "parser"
extra_state_tokens = false
hidden_width = 128
maxout_pieces = 3
use_upper = true
nO = null

[components.parser.model.tok2vec]
@architectures = "spacy.Tok2VecListener.v1"
width = ${components.tok2vec.model.encode.width}
upstream = "*"

[components.tagger]
factory = "tagger"

[components.tagger.model]
@architectures = "spacy.Tagger.v1"
nO = null

[components.tagger.model.tok2vec]
@architectures = "spacy.Tok2VecListener.v1"
width = ${components.tok2vec.model.encode.width}
upstream = "*"

[components.tok2vec]
factory = "tok2vec"

[components.tok2vec.model]
@architectures = "spacy.Tok2Vec.v1"

[components.tok2vec.model.embed]
@architectures = "spacy.MultiHashEmbed.v1"
width = ${components.tok2vec.model.encode.width}
attrs = ["ORTH","SHAPE"]
rows = [5000,2500]
include_static_vectors = false

[components.tok2vec.model.encode]
@architectures = "spacy.MaxoutWindowEncoder.v1"
width = 96
depth = 4
window_size = 1
maxout_pieces = 3

[corpora]

[corpora.dev]
@readers = "spacy.Corpus.v1"
path = ${paths.dev}
max_length = 0
gold_preproc = false
limit = 0
augmenter = null

[corpora.train]
@readers = "spacy.Corpus.v1"
path = ${paths.train}
max_length = 2000
gold_preproc = false
limit = 0
augmenter = null

[training]
dev_corpus = "corpora.dev"
train_corpus = "corpora.train"
seed = ${system.seed}
gpu_allocator = ${system.gpu_allocator}
dropout = 0.1
accumulate_gradient = 1
patience = 1600
max_epochs = 0
max_steps = 20000
eval_frequency = 200
frozen_components = []
before_to_disk = null

[training.batcher]
@batchers = "spacy.batch_by_words.v1"
discard_oversize = false
tolerance = 0.2
get_length = null

[training.batcher.size]
@schedules = "compounding.v1"
start = 100
stop = 1000
compound = 1.001
t = 0.0

[training.logger]
@loggers = "spacy.ConsoleLogger.v1"
progress_bar = false

[training.optimizer]
@optimizers = "Adam.v1"
beta1 = 0.9
beta2 = 0.999
L2_is_weight_decay = true
L2 = 0.01
grad_clip = 1.0
use_averages = false
eps = 0.00000001
learn_rate = 0.001

[training.score_weights]
dep_las_per_type = null
sents_p = null
sents_r = null
ents_per_type = null
dep_uas = 0.17
dep_las = 0.17
sents_f = 0.0
ents_f = 0.33
ents_p = 0.0
ents_r = 0.0
tag_acc = 0.33

[pretraining]

[initialize]
vectors = null
init_tok2vec = ${paths.init_tok2vec}
vocab_data = null
lookups = null

[initialize.components]

[initialize.tokenizer]
```

many thanks in advance
